### DISCUSSION TITLE
Looking for help with advanced LinkHints functionality

### DISCUSSION THREAD

**Discussion by ghost** [gdh1995/vimium-c#889] [Q&A] [answered]
I'm hoping to use j/k to select the next available link in LinkHints (similar to using tab) like this, and use `<S-L` to send a "Shift-Enter" keystroke to the browser (to open the selected link in a new window). Is this possible? I have tried using "ScrollSelect" but I cant figure out how to use it. 

Thank you if anyone can help, I am a huge fan of this extension :)

**Comment by gdh1995** [comment, marked as answer]
The suggested way is:
1. firstly, enable `Use the link's name and characters for link-hint filtering` on Vimium C Options page
2. then you may use `Tab` and `Shift+Tab` in LinkHints mode, and a "current" mark will be in not black but gray.

If you do want the 3 keys, you may use `mapKey` to achieve it:
```
mapKey <j:l> <tab>
mapKey <k:l> <s-tab>
mapKey <l:l> <enter>
mapKey <L:l> <s-enter>
```

See https://github.com/gdh1995/vimium-c/wiki/Use-in-another-keyboard-layout#per-mode-mapkey
